packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1600,6 +1600,25 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
 
   bool _debugLocked = false; // used to prevent re-entrant calls to push, pop, and friends
 
+  bool _shouldAbandonInitialRoute(List<Route<dynamic>> plannedInitialRoutes) {
```

Comment:
This method would probably benefit from a quick comment explaining when and why we want to abandon a route.

Reply:
Since I'm making this change for all platforms, this method isn't necessary anymore.